python/mlcroissant/recipes/tfds_croissant_builder.ipynb

```diff
@@ -46,12 +46,13 @@
    "outputs": [],
    "source": [
     "%%capture --no-display\n",
-    "# Install mlcroissant from the source\n",
-    "!pip install \"git+https://github.com/${GITHUB_REPOSITORY:-mlcommons/croissant}.git@${GITHUB_HEAD_REF:-main}#subdirectory=python/mlcroissant&egg=mlcroissant[dev]\"\n",
+    "# Install mlcroissant\n",
+    "!pip install mlcroissant\n",
```

Comment:
This breaks the feature that the notebook is run in the GitHub Actions on the current branch to test it end-to-end. What doesn't work with the current approach?

Reply:
Mmmh looks like the execution times out. Probably GitHub has some issues right now?

Reply:
It also times out in the said GitHub Actions: https://github.com/mlcommons/croissant/actions/runs/8556498478/job/23446447330 (currently broken on main)

Reply:
The command works locally so that sounds more like an issue with Colab

Reply:
if its a temporary timeout issue we can ignore i think

it also affects this notebook https://colab.research.google.com/github/mlcommons/croissant/blob/main/python/mlcroissant/recipes/introduction.ipynb which is linked directly to colab here https://github.com/mlcommons/croissant/tree/main/python/mlcroissant

![Screenshot from 2024-04-05 12-19-07](https://github.com/mlcommons/croissant/assets/26168435/b7e7823f-43d5-45db-a0e2-062869ccb644)

if we expect users to inspect the recipes in browser through colab, another fix would be to use `||` in bash to catch install with `pip install mlcroissant` in case these timeouts occur

`!pip install "git+https://github.com/${GITHUB_REPOSITORY:-mlcommons/croissant}.git@${GITHUB_HEAD_REF:-main}#subdirectory=python/mlcroissant&egg=mlcroissant[dev]" || pip install mlcroissant`

Reply:
OK, so `%%capture --no-display` was obfuscating the actual errors. I'm trying a fix that would work for both humans and the CI: https://github.com/mlcommons/croissant/pull/630

Reply:
works great ! closing this for #630